13_iv_antibiotics_pharmacy.sql

```diff
@@ -0,0 +1,87 @@
+CREATE OR REPLACE TABLE `physionet.iv_abx_pharmacy` as
+
+select 
+subject_id,
+hadm_id,
+stay_id,
+DATETIME_TRUNC(starttime, HOUR) as chart_hour,
+intime,
+outtime,
+medication,
+min(entertime) as entertime,
+min(starttime) as starttime,
+min(route) as route,
+max(stoptime) as stoptime,
+min(frequency) as frequency,
+min(disp_sched) as disp_sched,
+min(doses_per_24_hrs) as doses_per_24_hrs,
+min(duration) as duration,
+min(duration_interval) as duration_interval,
+count(*) as records
```

Comment:
Nit: Try using `count(1)` instead - it will give a performance boost. The problem is that `count(*)` loads all the data from the table, while `count(1)` just loads the number 1 in place of the data. It's just counting existence, so it's equivalent.